### PR TITLE
[dev/v2] Update `ruff` to `0.1.14`

### PR DESCRIPTION
**Pull request by FirelightFlagboy**

### CODE REVIEW

**Comment by touilleMan** [comment]
Wait for `dev/v3` merge before merging this

**Comment by FirelightFlagboy** [comment]
@touilleMan note that the PR target `dev/v2`

**Comment by touilleMan** [comment]
> @touilleMan note that the PR target dev/v2

Ha ! my bad !

Then I'm all for this PR :+1:

**Review by touilleMan — APPROVED** [review]

**Pull request merged**
